man/systemd-cryptenroll.xml

```diff
@@ -251,55 +256,65 @@
           <!-- See: https://github.com/tianocore-docs/edk2-TrustedBootChain/blob/main/4_Other_Trusted_Boot_Chains.md -->
           <!-- See: https://wiki.archlinux.org/title/Trusted_Platform_Module#Accessing_PCR_registers -->
 
-          <tgroup cols='2' align='left' colsep='1' rowsep='1'>
+          <tgroup cols='3' align='left' colsep='1' rowsep='1'>
             <colspec colname="pcr" />
+            <colspec colname="string_alias" />
             <colspec colname="definition" />
 
             <thead>
               <row>
                 <entry>PCR</entry>
+                <entry>alias</entry>
                 <entry>Explanation</entry>
               </row>
             </thead>
 
             <tbody>
               <row>
                 <entry>0</entry>
+                 <entry>core-firmware</entry>
                 <entry>Core system firmware executable code; changes on firmware updates</entry>
               </row>
 
               <row>
                 <entry>1</entry>
+                <entry>platform-config</entry>
                 <entry>Core system firmware data/host platform configuration; typically contains serial and model numbers, changes on basic hardware/CPU/RAM replacements</entry>
               </row>
 
               <row>
                 <entry>2</entry>
+                <entry>external-code</entry>
                 <entry>Extended or pluggable executable code; includes option ROMs on pluggable hardware</entry>
               </row>
 
               <row>
                 <entry>3</entry>
+                <entry>external-firmware</entry>
```

Comment:
external-configuration actually. The spec says:

> 3.3.4.4 PCR[3] – EFI Driver/Application Configuration
> This section contains the PCR[3] measurement requirements for a UEFI platform. In general, entities measure into PCR[3] data that is associated with code modules that are measured into PCR[2]. For example, on a UEFI platform, this includes but is not limited to UEFI variables defined and managed by drivers. See section 10.2.4 Measuring Industry Standard Tables and Data Structures. An example of information measured into PCR[3] is a SCSI controller’s configuration of its hard disks, e.g., RAID type, drive assignments, etc.

Reply:
> external-configuration actually. The spec says:
> 
> > 3.3.4.4 PCR[3] – EFI Driver/Application Configuration
> > This section contains the PCR[3] measurement requirements for a UEFI platform. In general, entities measure into PCR[3] data that is associated with code modules that are measured into PCR[2]. For example, on a UEFI platform, this includes but is not limited to UEFI variables defined and managed by drivers. See section 10.2.4 Measuring Industry Standard Tables and Data Structures. An example of information measured into PCR[3] is a SCSI controller’s configuration of its hard disks, e.g., RAID type, drive assignments, etc.

There's been no debate on PCR[3], the current option was totally from me. 
So it's switching swiftly to "external-configuration"

Thank you so much for the guidance.